src/utilities/getIntrospectionQuery.ts

```diff
@@ -52,6 +61,7 @@ export function getIntrospectionQuery(options?: IntrospectionOptions): string {
     schemaDescription: false,
     inputValueDeprecation: false,
     oneOf: false,
+    typeDepth: 9,
```

Comment:
so this default is a breaking change. If we want this to land in v16 then we should do `Infinity` and in `v17` we can restrict the limits.

Reply:
@saihaj I think you may have misunderstood which depth this relates to - it's the `ofType { ofType { ofType { ... } } }` depth of the generated query. We couldn't set that to infinity.